src/core/services/ripple/ripple.js

```diff
@@ -432,6 +432,7 @@ InkRippleCtrl.prototype.removeRipple = function (ripple) {
   if (index < 0) return;
   this.ripples.splice(this.ripples.indexOf(ripple), 1);
   ripple.removeClass('md-ripple-active');
+  ripple.addClass('md-ripple-remove');
```

Comment:
Shouldn't this class be removed at some point? Otherwise it will always stick around after the first click.